omero/omero-monitoring-agents.yml

```diff
@@ -32,6 +32,7 @@
   roles:
 
   - role: ome.omero_web_django_prometheus
+    when: 'not (omero_web_python3 | default(False))'
```

Comment:
```suggestion
```
https://travis-ci.org/ome/prod-playbooks/jobs/637371128 failed because this was disabled but the tests were still attempting to test prometheus. The role has been updated so this should now work.